app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="<%= I18n.locale %>">
```

Comment:
noticed this issue in the same report that identified the hreflang issues. this mismatch can also cause problems.